source/SIL.AppBuilder.Portal.Backend.Tests/Acceptance/APIControllers/UsersTest.cs

```diff
@@ -118,5 +120,26 @@ public async Task Patch_SomeUser_Different_Organization_NotFound()
 
             Assert.Equal(HttpStatusCode.NotFound, response.StatusCode);
         }
+
+        /// <summary>
+        /// Getting the current user (edit the user profile) when a different
+        /// organization is the current organization (e.g. viewing users from
+        /// a different organization) should always be allowed and return the
+        /// current user.
+        /// </summary>
+        /// <returns>The current user different organization found.</returns>
+        [Fact]
+        public async Task Get_CurrentUser_Different_Organization_Found()
```

Comment:
This test name should be more meaningful.
maybe something like `Get_User_WhenIdMatchesTheCurrentUser_OrganizationIdDoesNotMatter()` or something to that effect.

Reply:
`Get_CurrentUser_CurrentOrganizationDoesNotMatter_Found`

Reply:
it's not the CurrentUser endpoint though.
I guess I've (maybe) been doing

`{HTTP Verb}_{Action}_{Context?}_{Assertion}`